vmdb/app/models/mixins/miq_provision_mixin.rb

```diff
@@ -66,10 +66,20 @@ def get_owner
       email = get_option(:owner_email).try(:downcase)
       return if email.blank?
       requester = miq_request.requester
-      requester.email.to_s.downcase == email ? requester : User.where("lower(email) = ?", email).first
+      owner = requester.email.to_s.downcase == email ? requester : User.where("lower(email) = ?", email).first
+      set_owner_group(owner)
```

Comment:
@kbrock What do you think about using the tap method here?  `owner.tap {|o| set_owner_group(o)}`

Then the `set_owner_group` method does not have to be concerned with returning the user object, which seems a little odd anyway, and could change to be something like this:

```ruby
def set_owner_group(user)
  group_id = get_option(:owner_group)
  return if user.nil? || group_id.nil?

  group = user.miq_groups.detect { |g| g.description == group_id }
  user.current_group = group if group
end
```